tests/puppeteer/campaigns/functional/BO/03_catalog/02_categories/07_exportCategories.js

```diff
@@ -0,0 +1,76 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const CategoriesPage = require('@pages/BO/catalog/categories');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_catalog_categories_helperCard';
```

Comment:
```suggestion
const baseContext = 'functional_BO_catalog_categories_exportCategeories';
```

Reply:
```suggestion
const baseContext = 'functional_BO_catalog_categories_exportCategories';
```
Be careful there's a typo in the suggestion by Nesrine

Reply:
done